include/platform/mir/input/event_builder.h

```diff
@@ -38,6 +38,7 @@ class EventBuilder
 public:
     EventBuilder() = default;
     virtual ~EventBuilder() = default;
+    /// Timestamps in returned events are automatically calibrated to the Mir clock
     using Timestamp = std::chrono::nanoseconds;
```

Comment:
It is rather a shame to introduce another `Timestamp` alias. (C.f. `mir/time/types.h`).

Particularly, as elsewhere this is known as `mir::time::Duration`

Reply:
Maybe this could use some sorting out. The values being represented are timestamps, not durations, so using that alias would be incorrect. `mir::Timestamp` should probably be used across Mir, but that refactor is out of scope here.